resources/lang/en/book.php

```diff
@@ -22,5 +22,9 @@
     'image' => 'Image',
     'submit' => 'Submit',
     'reset' => 'Reset',
-    'back' => 'Back'
+    'back' => 'Back',
+    'confirm' => 'Confirm',
+    'do_you_want_to_delete_this_book' => 'Do you want to delete this book',
```

Comment:
do_you_want_to_delete_this_book
chỗ này nên sửa lại là:
'confirm' => [
'delete' => 'Do you want to delete this book'
]